src/xrpld/rpc/detail/TransactionSign.cpp

```diff
@@ -1150,7 +1227,7 @@ transactionSubmitMultiSigned(
     }
 
     // Grind through the JSON in tx_json to produce a STTx.
-    std::shared_ptr<STTx> stpTrans;
+    std::shared_ptr<STTx> stTx;
```

Comment:
Please avoid Hungarian notation

Reply:
This isn't hungarian notation, just `STTx` in `camelCase`.